runelite-client/src/main/java/net/runelite/client/plugins/idlenotifier/IdleNotifierPlugin.java

```diff
@@ -90,6 +92,7 @@
 	private int lastCombatCountdown = 0;
 	private Instant sixHourWarningTime;
 	private boolean ready;
+	private Instant lastInteractingFishing;
```

Comment:
Maybe rename this to lastInteractingNonCombat to generalize this a bit for future use? Or even better, simply reuse lastInteracting and add like boolean inCombat to differentiate between out of combat and non-combat notification, as otherwise the logic is just duplicated